src/ui_elements/mainmenu.gd

```diff
@@ -13,60 +13,93 @@ onready var server_menu: Control = $Server
 # --Interface--
 func _ready():
 	menu = MenuType.MAIN
-	set_Visible_Menu(menu)
+	setVisibleMenu(menu)
 
 func _input(event):
 	if event.is_action_pressed("ui_accept"):
-		join_Event(menu)
+		joinEvent(menu)
 
 func _on_Back_pressed():
 	menu = MenuType.MAIN
-	set_Visible_Menu(menu)
+	setVisibleMenu(menu)
 
 func _on_GameJoinButton_pressed() -> void:
 	menu = MenuType.JOIN
-	set_Visible_Menu(menu)
+	setVisibleMenu(menu)
 
 func _on_GameCreateButton_pressed() -> void:
 	menu = MenuType.CREATE
-	set_Visible_Menu(menu)
+	setVisibleMenu(menu)
 
 func _on_ServerStartButton_pressed() -> void:
 	menu = MenuType.SERVER
-	set_Visible_Menu(menu)
+	setVisibleMenu(menu)
 
 func _on_AppQuitButton_pressed() -> void:
 	get_tree().quit()
 
-func set_Visible_Menu(menu) -> void:
-	hide_Menus()
+func setVisibleMenu(menu) -> void:
+	hideMenus()
 	match menu:
 		MenuType.MAIN: main_menu.visible = true
 		MenuType.JOIN: join_menu.visible = true
 		MenuType.CREATE: create_menu.visible = true
 		MenuType.SERVER: server_menu.visible = true
 
-func hide_Menus() -> void:
+func hideMenus() -> void:
 	main_menu.visible = false
 	join_menu.visible = false
 	create_menu.visible = false
 	server_menu.visible = false
 
 # --Backend--
-func join_Event(menu):
+func joinGame() -> void:
+	var nameField: LineEdit = $Join/Name
+	var serverField: LineEdit = $Join/Address
+	var port: int = 46690
+	var host: String = serverField.text
+	var name: String = nameField.text
+	var cut_pos: int = host.find(":")
+	if cut_pos != -1:
+		port = int(host.right(cut_pos))
+		host = host.left(cut_pos)
+	print_debug("port: ", port, ", host: ", host)
+	if host == "" or name == "":
+		return
+	Connections.joinGame(host, port, name)
+
+func createGame() -> void:
+	var nameField: LineEdit = $Create/Name
+	var portField: LineEdit = $Create/Port
+	var port: int = int(portField.text)
+	var name: String = nameField.text
+	if name == "":
+		return
+	Connections.createServer(port, name)
```

Comment:
This function, createGame, ends up calling createServer on Connections, breaking the naming scheme the other two similar functions had.

eg. createDedicated had Connections.createDedicated
joinGame had Connections.joinGame

I'd suggest changing the function it is calling to be called Connections.createGame.

Reply:
good point, thanks, corrected.